### ISSUE TITLE
Move keys to web-based admin control panel

### ISSUE THREAD

**Issue by mvl22** [cyclestreets/cyclescape#491]
Looking at the settings at:
https://github.com/cyclestreets/cyclescape-chef/tree/master/data-bags/secrets

including the e-mail parameters, I wonder whether all this can be moved into a control panel based arrangement. As far as I can see, none of them are actually required at setup time.

Once set up, the site should then be in a state of not running until the control panel (being the only section that would be enabled) has these filled in. The person setting up the site would then have to enter them. As long as that is only run if a database connection is properly established, I can't see any security risk here.

**Comment by mvl22** [comment]
Is this now done @nikolai-b?

**Comment by nikolai-b** [comment]
No, we have the site config but moving the DNS, Rollbar etc. keys is not easy as they are loaded when rails boots up.  I can investigate changing them but we'd also need to restart all the workers when they are changed.
